Source/Factory/Internal/NSInvocation+TCFWrapValues.h

```diff
@@ -17,4 +17,7 @@
 /** Get argument at index, wrapping primitive values in NSValue if needed */
 - (id)typhoon_getArgumentObjectAtIndex:(NSInteger)idx;
 
+/** Get return value, wrapping primitive values in NSValue if needed*/
+- (id)typhoon_getReturnValue;
```

Comment:
What's the use case for that? I don't think Typhoon supports definitions for primitive values. And we also can't support injecting primitive values. Can you advice where its used?